doc/content/verification_validation_examples/slab_benchmark/slab_benchmark.md

```diff
@@ -181,21 +181,21 @@ Substituting [eqn:flux_solution] and [eqn:temperature] into [eqn:constraint] giv
   \label{eqn:power_condition}
 \end{equation}
 
-The example set of model parameter values shown in [tab:params] are the suggested benchmark parameters [!citep](griesheimer2022analytical), resulting in $k_\text{eff} = 0.29557$ and a heated slab length of $L = 106.47$ cm with $P = 1.0\times10^{22}$ eV/s.
+The example set of model parameter values shown in [tab:params] are the suggested benchmark parameters [!citep](griesheimer2022analytical), resulting in $k_\text{eff} = 0.29557$ and a heated slab length of $L = 106.47$ `cm` with $P = 1.0\times10^{22}$ `eV/s`.
```

Comment:
General comment: I do not think you need the "`" signs around units. We usually do not use them in papers and in the rest of the documentation. For consistency, I recommend removing them.